src/components/tabs/tabs.component.js

```diff
@@ -42,10 +42,9 @@ const Tabs = ({
   ...rest
 }) => {
   /** The children nodes converted into an Array */
-  const filteredChildren = useMemo(
-    () => Children.toArray(children).filter((child) => child),
-    [children]
-  );
+  const filteredChildren = useMemo(() => Children.toArray(children), [
```

Comment:
is there a reason we're no longer filtering to remove any falsy children? `null` is a valid child and will certainly cause an error when accessing `child.props` below - and this could happen in practice.

If we are sure we don't need to filter then we should change the name to something like `childrenArray`, since we're no longer filtering. Also we could probably do without the array and just use `Children.map` later when we map over the children.

But unless I'm missing something important I think we should keep this filter in place.

Reply:
Likewise I think we still need to filter for falsy children here unless I'm missing something somewhere

Reply:
Ah this was an oversight on my part, I didn't realise this removed falsy children 👍🏼